docs/finalproject/prep.md

```diff
@@ -23,10 +23,10 @@ Getting the Final Projects ready is not a task that can be done a week before th
 - [ ] Let volunteers choose which project they would like to be on
 
 **When you are on SQL Module**
-- [ ] Trainees must have finalised the Full Stack assessment 
+- [ ] Trainees' must have completed up to Level 300 of the Full Stack assessment by week 3 of the module before Final Projects and finalised 
```

Comment:
```suggestion
- [ ] Trainees must have completed up to Level 300 of the [Full Stack Assessment](https://github.com/CodeYourFuture/Table-of-Contents) by week 3 of the module before Final Projects.
```

This is plural, but not plural possessive, in this case. It was the other use of trainees in your previous PR that was plural possessive. It's a minefield! 😅